### PR TITLE
html-movie-seat-booking

### PR DESCRIPTION
**Pull request by zalevskyi**
### html-movie-seat-booking

[Demo](https://zalevskyi.github.io/kottans-frontend/practice/html-movie-seat-booking/)

Please, review.

### CODE REVIEW

File: submissions/zalevskyi/html-movie-seat-booking/app.css [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    line-height: 1em;
+}
+.screen {
+    width: 75vw;
+    margin: 20px auto 5vh auto;
+    box-shadow: 0px 5px 0px lightskyblue;
+    text-transform: uppercase;
+    text-align: center;
+}
+.seat {
```
Comment (A-Ostrovnyy) [inline submissions/zalevskyi/html-movie-seat-booking/app.css:32]: Add styles on `:hover`, `:active`, and `:focus` states. Also, add `cursor: pointer;`

File: submissions/zalevskyi/html-movie-seat-booking/app.css [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    border: none;
+    text-align: center;
+}
+.book__wrapper {
+    margin: 50px;
+    padding: 0;
+    border: none;
+    text-align: center;
+}
+.book__button {
```
Comment (A-Ostrovnyy) [inline submissions/zalevskyi/html-movie-seat-booking/app.css:83]: Same as at line 32

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
See my comment below!

**Comment by zalevskyi** [comment]
Thank you for the review!
All recommendations are implemented (Add styles on :hover, :active, and :focus states. Also, add cursor: pointer;)
Please review once more.

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**